test/browser.core.test.js

```diff
@@ -25,110 +41,112 @@ describe('options.captureUncaught', function () {
 
-      var body = JSON.parse(server.requests[0].requestBody);
+    server.respond();
 
-      expect(body.data.body.trace.exception.message).to.eql('test error');
-      expect(body.data.notifier.diagnostic.raw_error.message).to.eql(
-        'test error',
-      );
-      expect(body.data.notifier.diagnostic.is_uncaught).to.eql(true);
+    const body = JSON.parse(server.requests[0].requestBody);
 
-      // karma doesn't unload the browser between tests, so the onerror handler
-      // will remain installed. Unset captureUncaught so the onerror handler
-      // won't affect other tests.
-      rollbar.configure({
-        captureUncaught: false,
-      });
+    expect(body.data.body.trace.exception.message).to.eql('test error');
+    expect(body.data.notifier.diagnostic.raw_error.message).to.eql(
+      'test error',
+    );
+    expect(body.data.notifier.diagnostic.is_uncaught).to.be.true;
 
-      done();
-    }, 1);
+    // karma doesn't unload the browser between tests, so the onerror handler
+    // will remain installed. Unset captureUncaught so the onerror handler
+    // won't affect other tests.
+    rollbar.configure({
+      captureUncaught: false,
+    });
   });
 
-  it('should respond to enable/disable in configure', function (done) {
-    var server = window.server;
-    var element = document.getElementById('throw-error');
+  it('should respond to enable/disable in configure', async function () {
+    const server = window.server;
+    const element = document.getElementById('throw-error');
+    expect(server).to.exist;
+    expect(element).to.exist;
+
     stubResponse(server);
     server.requests.length = 0;
 
-    var options = {
+    const rollbar = (window.rollbar = new Rollbar({
       accessToken: 'POST_CLIENT_ITEM_TOKEN',
       captureUncaught: false,
-    };
-    var rollbar = (window.rollbar = new Rollbar(options));
+    }));
 
     element.click();
 
-    setTimeout(function () {
-      server.respond();
-      expect(server.requests.length).to.eql(0); // Disabled, no event
-      server.requests.length = 0;
+    await setTimeout(1);
 
-      rollbar.configure({
-        captureUncaught: true,
-      });
+    server.respond();
+    expect(server.requests.length).to.eql(0); // Disabled, no event
+    server.requests.length = 0;
 
-      element.click();
+    rollbar.configure({
+      captureUncaught: true,
+    });
 
-      setTimeout(function () {
-        server.respond();
+    element.click();
+
+    await setTimeout(1);
 
-        var body = JSON.parse(server.requests[0].requestBody);
+    server.respond();
 
-        expect(body.data.body.trace.exception.message).to.eql('test error');
-        expect(body.data.notifier.diagnostic.is_anonymous).to.not.be.ok;
+    const body = JSON.parse(server.requests[0].requestBody);
 
-        server.requests.length = 0;
+    expect(body.data.body.trace.exception.message).to.eql('test error');
+    expect(body.data.notifier.diagnostic.is_anonymous).to.be.undefined;
```

Comment:
`to.not.be.ok` ≠ `to.be.undefined`. Chai 5 adds more precise checks for these situations.